kaggle_environments/envs/kore_fleets/helpers.py

```diff
@@ -122,10 +122,18 @@ def from_str(raw: str):
         if not raw:
             return None
         if raw.startswith(ShipyardActionType.SPAWN.name):
-            return ShipyardAction.spawn_ships(int(raw.split("_")[1]))
+            _, ship_str = raw.split("_")
+            try:
+                num_ships = int(ship_str.split(".")[0])
+            except _:
```

Comment:
Should we log on exception here?

Did you consider just parsing as a double and then converting to int?